CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.5.0] - 2016-04-18
+
+### Fixed
+- Static dependency libraries will no longer be linked to Leatherman consumers
+- Interfacing with ruby APIs for 64-bit integers on Windows
```

Comment:
This is a breaking change, as it removes rb_num2long and rb_num2ulong. They were dangerously inconsistent to use on Windows.